source/reference/footnotes.txt

```diff
@@ -0,0 +1,25 @@
+=========
+Footnotes
+=========
+
+Limit footnotes to tangential explanations of greater technical depth that build
+understanding but can be ignored.
```

Comment:
```suggestion
Footnotes allow you to explain topics in greater detail,  but can be ignored.
```
Suggestion: I think this can be simplified to something similar to the suggestion above-- as is, its a bit wordy and tough to understand.